src/vcsparser.core/DailyCodeChurnBugDatabase.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace vcsparser.core
+{
+    public class DailyCodeChurnBugDatabase
+    {
+        public int NumberOfChanges { get; set; }
+
+        public int NumberOfChangesWithFixes { get; set; }
+
+        public int Added { get; set; }
+
+        public int Deleted { get; set; }
+
+        public int ChangesBefore { get; set; }
+
+        public int ChangesAfter { get; set; }
+
+        public int TotalLinesChanged {

```

Comment:
I think it is a good idea to rename this to TotalLinesChangedInFixes since everything that is in the bug database is bug fixes.

Reply:
Renamed in 436638d125bb40b11223df403d8dd9b234d55800